FLAME_GPU_2_python_tutorial.ipynb

```diff
@@ -49,13 +49,16 @@
    },
    "outputs": [],
    "source": [
-    "import sys\n",
-    "!{sys.executable} -m pip install -I https://github.com/FLAMEGPU/FLAMEGPU2/releases/download/v2.0.0-alpha.1/pyflamegpu-2.0.0a1+cuda112-cp38-cp38-linux_x86_64.whl\n",
+    "import importlib.util\n",
+    "if importlib.util.find_spec('pyflamegpu') is None:\n",
+    "    import sys\n",
+    "    !{sys.executable} -m pip install -I https://github.com/FLAMEGPU/FLAMEGPU2/releases/download/v2.0.0-rc/pyflamegpu-2.0.0rc0+cuda112-cp38-cp38-linux_x86_64.whl # type: ignore\n",
+    "\n",
+    "import pyflamegpu\n",
```

Comment:
pyflamegpu is being imported twice in this cell now, coudl just do with the imports tidying a little.